hack/release.sh

```diff
@@ -11,7 +11,9 @@ echo "$REGISTRY_PASSWORD" | ko login -u "$REGISTRY_USERNAME" --password-stdin "$
 
 echo "Building container image"
 
+echo "Adding io.shipwright.vcs-ref label with value: ${GITHUB_SHA}"
+
 # Using defaults, this pushes to:
 # quay.io/shipwright/shipwright-build-controller:latest
-KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS}" ko resolve -t "$TAG" --bare --platform=all -R -f deploy/ > release.yaml
-KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS} -tags=pprof_enabled" ko resolve -t "$TAG-debug" --bare --platform=all -R -f deploy/ > release-debug.yaml
+KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS}" ko resolve -t "$TAG" --image-label "io.shipwright.vcs-ref=${GITHUB_SHA}" --bare --platform=all -R -f deploy/ > release.yaml
+KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" GOFLAGS="${GO_FLAGS} -tags=pprof_enabled" ko resolve -t "$TAG-debug" --image-label "io.shipwright.vcs-ref=${GITHUB_SHA}" --bare --platform=all -R -f deploy/ > release-debug.yaml
```

Comment:
Just making a note that we are using a label-schema inspired label, and not label-schema itself. To do so would require using the `org.label-schema` namespace and also supplying the `schema-version` label.

See https://label-schema.org/rc1/

Reply:
@adambkaplan good catch, indeed this is a label-schema inspired, not fully compliant with the docs there.